start.sh

```diff
@@ -0,0 +1,18 @@
+export DB_HOST=127.0.0.1
+export DB_NAME=bitcore-tbtg
+export BITCORE_CONFIG_PATH=$HOME/bitcore/tbtg.json
+export NETWORK=mainnet
+export CHAIN=BTG
+
+case $1 in
+node)
+  exec 1>debug.node 2>&1
+  npm run node
```

Comment:
Maybe pipe it to `tee` to allow screen output and file output?

Reply:
To make both work, need to use tee, yes, will fix it

Reply:
Oh, and should append to the log file instead of creating a new one